.buildkite/pipeline.yml

```diff
@@ -46,7 +46,7 @@ steps:
     command: make clean install-vendor docker-integration-test
     env:
       CGO_ENABLED: 0
-      GIMME_GO_VERSION: 1.10.x
+      GIMME_GO_VERSION: 1.12.x
```

Comment:
Curious: is there any reason keeping `make clean install-vendor docs-test` on 1.10?